### PR TITLE
Update Bootstrappers

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
This repo got missed when updating bootstrappers.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+__get_remote_file() {
+    local remote_path=$1
+    local local_path=$2
+
+    if [[ "$remote_path" != 'http'* ]]; then
+        cp "$remote_path" "$local_path"
+        return 0
+    fi
+
+    local failed=false
```
Comment (dougbu) [inline run.sh]: Would be simpler if `$failed` were initially `true` or this was `$success`.

File: run.sh [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        fi
+    elif __machine_has python ; then
+        if python -c "import json,codecs;obj=json.load(codecs.open('$config_file', 'r', 'utf-8-sig'))" >/dev/null ; then
+            config_channel="$(python -c "import json,codecs;obj=json.load(codecs.open('$config_file', 'r', 'utf-8-sig'));print(obj['channel'] if 'channel' in obj else '')")"
+            config_tools_source="$(python -c "import json,codecs;obj=json.load(codecs.open('$config_file', 'r', 'utf-8-sig'));print(obj['toolsSource'] if 'toolsSource' in obj else '')")"
+        else
+            __warn "$config_file is invalid JSON. Its settings will be ignored."
+        fi
+    else
+        __warn 'Missing required command: jq or pyton. Could not parse the JSON file. Its settings will be ignored.'
```
Comment (dougbu) [inline run.sh]: typo: pyt**h**on

File: build/dependencies.props [hunk 3]
```diff
@@ -9,6 +9,6 @@
     <NETStandardLibraryNETFrameworkVersion>2.0.0-*</NETStandardLibraryNETFrameworkVersion>
     <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0-*</RuntimeFrameworkVersion>
     <TestSdkVersion>15.3.0</TestSdkVersion>
-    <XunitVersion>2.3.0-beta4-build3742</XunitVersion>
+    <XunitVersion>2.3.0</XunitVersion>
```
[the remark this answers is not in the document]
Comment (ryanbrandenburg) [reply build/dependencies.props]: Our other repos are also using 2.3.0. When we decide to update to whatever version, we should do so in every repo at once unless there's a compelling reason not to.

**Comment by ryanbrandenburg** [comment]
I filed aspnet/BuildTools#449 to address this feedback in all repos at once.

**Pull request merged**